projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx

```diff
@@ -142,8 +142,8 @@ export class DotDateRangeComponent {
             mode: 'range',
             altFormat: this.displayFormat,
             altInput: true,
-            maxDate: this.parseDate(this.max),
-            minDate: this.parseDate(this.min),
+            maxDate: this.max ? this.parseDate(this.max) : null,
+            minDate: this.min ? this.parseDate(this.min) : null,
```

Comment:
test for this change?

Reply:
no idea how to test this, I mean this is added to remove the noise that's printed on the terminal when running unit test.

With or without this, there's not a difference in the DOM, so I have no idea how I could so any assert on this

Reply:
Can you add a little range min and max, open the table, get all the <td> and check which one have a specific class?